content/pages/meetup/2023/0913-nycu.rst

```diff
@@ -0,0 +1,93 @@
+========================================
+Meetup 2023 September 13th in NYCU
+========================================
+
+:date: 2023-09-08 15:00
+:url: meetup/2023/0913-nycu
+:save_as: meetup/2023/0913-nycu.html
+
+Date
+-----
+
+* Date: September 13th, 2023
+* Time: 18:30 -- 21:30 (3 hours)
+
+|
+ 
+Agenda
+--------
+
+* 18:30-19:30 Free discussion
+* 19:30-20:00 Reviewing BLOOM, the Large Language Model
+* 20:00-21:30 Preparational meeting for September scisprint 
+
+|
+
+Subjects
+------------------
+
+Reviewing BLOOM, the Large Language Model
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+Speaker : Terry - A graduate student in the field of Biomedical Engineering at NYCU. 
+His research interests include bioinformatics, IoT applications, and software system development.
+
+In this sharing, Terry will provide a brief overview of the large language model, BLOOM. 
+The main topics will include how to train a large language model like GPT-3, the costs associated with training such a model, and the factors that affect training throughput.
+
+
+Preparational meeting for September scisprint
+++++++++++++++++++++++++++++++++++++++++++++++++
+
+Additionally, we will discuss the following items for preparing September scisprint. 
+
+* Task arrangement for volunteers
+* Promotion plan for September scisprint
+
```

Comment:
Please help me add the link of scisprint and kkitx. The sentance will look like : 
Please check September scisprint (https://sciwork.dev/sprint/2023/09-hsinchu) and kktix (https://sciwork.kktix.cc/events/scisprint-202309-hsinchu) for more information.

Reply:
Please use hyberlink and only display September scisprint and kktix.

Reply:
Okay, I've made the changes.